packages/devtools-reps/src/object-inspector/components/ObjectInspectorItem.js

```diff
@@ -286,7 +293,8 @@ class ObjectInspectorItem extends Component<Props> {
                 setExpanded: this.props.setExpanded
               });
             }
-          : undefined
+          : undefined,
+        onDoubleClick: onExpressionDblClick
```

Comment:
I see that the `renderLabel` function this appears in has a few interesting things we could try:

1.  The `onClick` has a check for selection:  `if (Utils.selection.documentHasSelection()) {` ; maybe that could be of use?

2.  There is a `depth` check; maybe if `depth` is one, and there's more than one depth, we ignore on depth 0?

Reply:
Thanks @darkwing. I implemented a depth check for that